### PR TITLE
Revenue forecast adjustments

### PR DESCRIPTION
**Pull request by carloslopes**

### CODE REVIEW

File: app/presenters/revenue_forecast_presenter.rb [hunk 1]
```diff
@@ -6,8 +6,8 @@ class RevenueForecastPresenter
   # should only show forecasts from 2022 and beyond
   REVENUE_FORECAST_START_YEAR = 2022
 
-  def initialize(year)
-    @forecasts = RevenueForecastService.year_forecast(year)
+  def initialize(year, market)
+    @forecasts = RevenueForecastService.year_forecast(year, market)
     @forecasts.sort_by! { |h| h[:project].name }
```
Comment (GabrielRMuller) [inline app/presenters/revenue_forecast_presenter.rb:11]: I know this wasn't added in this PR, but could you add this last line in a separate method?

File: app/services/revenue_forecast_service.rb [hunk 2]
```diff
@@ -48,16 +49,16 @@ def project_forecast(project, period = nil)
...
   end
 
-  def self.year_forecast(year)
-    new.year_forecast(year)
+  def self.year_forecast(year, market = nil)
+    new.year_forecast(year, market)
   end
 
-  def year_forecast(year)
+  def year_forecast(year, market = nil)
```
Comment (GabrielRMuller) [inline app/services/revenue_forecast_service.rb:56]: do we need `market = nil` in here since we already have it on the class method? (line 52)

**Review by GabrielRMuller — APPROVED** [review]

File: app/services/revenue_forecast_service.rb (hunk 2 above)
Comment (carloslopes) [reply app/services/revenue_forecast_service.rb:56]: We don't. I'll fix this

File: app/presenters/revenue_forecast_presenter.rb (hunk 1 above)
Comment (carloslopes) [reply app/presenters/revenue_forecast_presenter.rb:11]: Sure!

**Review by GabrielRMuller — APPROVED** [review]

**Pull request merged**